Jenkinsfile_k8s

```diff
@@ -1,18 +1,170 @@
-#!/usr/bin/env groovy
+def recordDeployment(owner, repo, ref, status, environmentURL, environment = "preview", description = "Deploy to preview environment") {
+  withCredentials([usernamePassword(credentialsId: 'github-app-infra', usernameVariable: 'GITHUB_APP', passwordVariable: 'GH_TOKEN')]) {
+    def json = writeJSON(returnText: true, json: [
+        "ref": ref,
+        "environment": environment,
+        "description": description,
+        "required_contexts": [],
+        "auto_merge": false,
+        "auto_inactive": false,
+        "transient_environment": false,
+    ])
+    def id = readJSON(text: sh(script: "gh api repos/${owner}/${repo}/deployments  -X POST --input - << EOF\n${json}\nEOF", returnStdout: true).trim()).id
+    if (id == ''){
+      error('Unable to create deployment')
+    }
+    json = writeJSON(returnText: true, json: [
+        "state": status,
+        "environment": environment,
+        "description": description,
+        "log_url": "${BUILD_URL}/console",
```

Comment:
not useful imo I think it's what is adding 2 clicks, normally you can just click 'view deployment' and it takes you straight there
```suggestion
```

if you want to keep it should at least remove the double /
```suggestion
        "log_url": "${BUILD_URL}console",
```

Reply:
Sure let's try it. It's all new to me